tree/ntuple/v7/src/RNTupleSerialize.cxx

```diff
@@ -731,7 +731,7 @@ RResult<std::uint32_t> ROOT::Experimental::Internal::RNTupleSerializer::Deserial
          return R__FAIL("feature flag buffer too short");
       bytes += DeserializeInt64(bytes, f);
       bufSize -= sizeof(std::int64_t);
-      flags.emplace_back(abs(f));
+      flags.emplace_back((f < 0) ? -f : f);
```

Comment:
Did you try std::abs? (well maybe it does not matter anyway).

Reply:
I thought it in the first place, but I think this is probably safer here.